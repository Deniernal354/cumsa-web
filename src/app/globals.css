@import "tailwindcss";

:root {
  --background: #f8fafc;
  --foreground: #0f172a;
  --surface: #ffffff;
  --muted: #64748b;
  --navy: #2a7487;
  --blue: #5aced8;
  --red: #c73221;
  --header-top: #123a66;
}

body {
  background: var(--background);
  color: var(--foreground);
  font-family: Arial, Helvetica, sans-serif;
}

a {
  text-underline-offset: 4px;
}

.link-cta {
  color: var(--blue);
  font-weight: 700;
  text-decoration: underline;
  text-underline-offset: 3px;
}

.link-cta:hover {
  color: var(--red);
}

.btn-cozy {
  display: inline-flex;
  align-items: center;
  justify-content: center;
  padding: 0.5rem 1rem;
  border-radius: 0.6rem;
  font-weight: 600;
  font-size: 0.9rem;
  transition:
    transform 150ms ease,
    box-shadow 150ms ease,
    opacity 150ms ease;
}

.btn-cozy:hover {
  transform: translateY(-1px);
  box-shadow: 0 6px 16px rgba(0, 0, 0, 0.08);
}

.btn-primary {
  background: var(--blue);
  color: white;
}

.btn-outline {
  background: var(--surface);
  color: var(--foreground);
  border: 1px solid rgba(0, 0, 0, 0.12);
}

:focus-visible {
  outline: 2px solid color-mix(in oklab, var(--blue) 70%, white);
  outline-offset: 2px;
}

.container-base {
  margin-left: auto;
  margin-right: auto;
  max-width: 80rem;
  padding-left: 1rem;
  padding-right: 1rem;
}

@media (min-width: 640px) {
  .container-base {
    padding-left: 1.5rem;
    padding-right: 1.5rem;
  }
}

@media (min-width: 1024px) {
  .container-base {
    padding-left: 2rem;
    padding-right: 2rem;
  }
}

.transition-base {
  transition: all 200ms ease-out;
}

.card {
  background: var(--surface);
  border: 1px solid rgba(0, 0, 0, 0.12);
  border-radius: 0.75rem;
}

.title {
  color: var(--red);
}

.tint-blue {
  filter: invert(28%) sepia(88%) saturate(3612%) hue-rotate(211deg) brightness(93%) contrast(98%);
}

.logo-mask-blue {
  background-color: var(--blue);
  -webkit-mask-image: url("/CU-MSA-LOGO-blank-2.png");
  mask-image: url("/CU-MSA-LOGO-blank-2.png");
  -webkit-mask-repeat: no-repeat;
  mask-repeat: no-repeat;
  -webkit-mask-position: center;
  mask-position: center;
  -webkit-mask-size: contain;
  mask-size: contain;
  display: inline-block;
}
