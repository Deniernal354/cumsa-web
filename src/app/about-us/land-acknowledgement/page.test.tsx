import { test, expect } from "@jest/globals";
import { render } from "@testing-library/react";
import LandAcknowledgementPage from "./page";

test("land acknowledgement page snapshot", () => {
  const { asFragment } = render(<LandAcknowledgementPage />);
  expect(asFragment()).toMatchSnapshot();
});
