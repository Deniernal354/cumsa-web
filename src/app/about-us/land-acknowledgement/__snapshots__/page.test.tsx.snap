// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`land acknowledgement page snapshot 1`] = `
<DocumentFragment>
  <div
    class="container-base py-12"
  >
    <h1
      class="text-3xl font-bold"
    >
      Land Acknowledgement
    </h1>
    <p
      class="mt-4 text-gray-700"
    >
      As Muslims committed to justice, we recognize that we live, learn, and worship on the unceded, unsurrendered territory of the Algonquin nation. We affirm our responsibility to support Indigenous self-determination and to work toward justice in word and deed.
    </p>
    <div
      class="mt-6 space-y-4 text-gray-700"
    >
      <p>
        We remember the Qurâ€™anic call to stand firm for justice (4:135) and the Prophetic teachings against oppression. We ask God to help us uphold these commitments and to work in solidarity with the Algonquin people.
      </p>
      <p>
        Ameen.
      </p>
    </div>
  </div>
</DocumentFragment>
`;
